src/App.js

```diff
@@ -18,6 +19,7 @@ function App() {
         <Route path="/new-project" element={<ProjectCreate />} />
         <Route path="/project/:id" element={<ProjectView />} />
         <Route path="/project/edit/:id" element={<ProjectEdit />} />
+        <Route path="/skills" element={<AddSkill />} />
```

Comment:
Try to be more consistent when you pick names for routes. So if we already go with a singular `/project` route, let's do the same for skill entity.

Reply:
Same for folder name, `skills` -> `skill`